### ISSUE TITLE
Trying to Implement a React App Feature Slider That uses Spotify API Audio Features to Generate a Playlist

### ISSUE THREAD

**Issue by emmyacosta** [soron93/selekta-client#1]
@numbing @angelascerbo @cyberdude @awicks44 @carmellemillar

HALLLLP! We me and @soron93,  are building our first react app for a class project 

I don't know where to start the basic idea is the user has to select the audio features through some sliders and it will  generate a playlist based on user input 
[****](https://whimsical.com/project-3-selekta-app-9TsBcDXDtm1cGAFzJhkqBs) Wire Frames 

Similar to this  - https://dubolt.com/

Basically the Spotify API has Audio Features 
https://developer.spotify.com/console/get-audio-features-several-tracks/ 
https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-audio-features

 Audio Features are ... 
danceability, tempo ,analog vs digital ect.

**Comment by angelascerbo** [comment]
@emmyacosta - you have something up and running, nice start! 🎉 

Which component are you working with?

**Comment by emmyacosta** [comment]
yeah ...  well thats the thing we want to connect the sliders to end points on the API soit can generate the list but not quite sure how thats done thanks @angelascerbo
